.travis.yml

```diff
@@ -1,37 +1,44 @@
-sudo: false
+os: linux
 language: scala
 stages:
   - name: test
   - name: release
     if: (branch = master AND type = push) OR (tag IS present)
 jobs:
   include:
-  - script: "./mill __.jvm[2.12.7].test"
+  - script: "./mill __.jvm[2.12.10].test"
     jdk: oraclejdk9
 
-  - script: "./mill __.jvm[2.13.0].test"
+  - script: "./mill __.jvm[2.13.1].test"
     jdk: openjdk10
 
-  - script: "./mill __.js[2.12.7].test"
+  - script: "./mill __.js[2.12.10,_].test"
     jdk: openjdk10
 
-  - script: "./mill __.js[2.13.0].test"
+  - script: "./mill __.js[2.13.1,_].test"
     jdk: openjdk10
 
+  - script: "./mill __.native[__].test"
+    jdk: openjdk8
+    before_install:
+      - curl https://raw.githubusercontent.com/scala-native/scala-native/master/scripts/travis_setup.sh | bash -x
+
   - script: "./mill all demo.fastOpt perftests.__.compile"
-    jdk: openjdk10
 
   - script: "./mill demo.fullOpt && sbt readme/run"
     jdk: oraclejdk9
 
   - stage: release
     script:
       - echo "$PGP_SECRET" | base64 --decode | gpg --import
-      - ./mill __[2.12.7].publish --sonatypeCreds "$SONATYPE_USERNAME:$SONATYPE_PASSWORD" --release true --gpgPassphrase "$PGP_PASSPHRASE"
+      - ./mill __[2.12.10].publish --sonatypeCreds "$SONATYPE_USERNAME:$SONATYPE_PASSWORD" --release true --gpgPassphrase "$PGP_PASSPHRASE"
 
   - stage: release
     script:
       - echo "$PGP_SECRET" | base64 --decode | gpg --import
-      - ./mill __[2.13.0].publish --sonatypeCreds "$SONATYPE_USERNAME:$SONATYPE_PASSWORD" --release true --gpgPassphrase "$PGP_PASSPHRASE"
-
+      - ./mill __[2.13.1].publish --sonatypeCreds "$SONATYPE_USERNAME:$SONATYPE_PASSWORD" --release true --gpgPassphrase "$PGP_PASSPHRASE"
```

Comment:
Given the concerns exposed [here](https://github.com/lihaoyi/fastparse/pull/239/files#r383546175) I think this will not continue to work for js because it seems to match only on modules having a single parameter.
Is it normal to have such a breaking change between 0.6.0-15 and 0.6.0-28 ?